src/domain/app/footer/Footer.tsx

```diff
@@ -16,9 +17,15 @@ const Footer: FunctionComponent = () => {
             <p>{t('footer.copyrightText')}</p>
           </div>
           <div className={styles.links}>
-            <a href="/accessibility">{t('accessibilityStatement.title')}</a> •{' '}
-            <a href="/terms">{t('termsOfService.title')}</a> •{' '}
-            <a href={t('descriptionOfTheFile.url')}>
+            <Link to="/accessibility#start">
+              {t('accessibilityStatement.title')}
```

Comment:
The `{  }` is quite bad tho, can we replace it with padding / margin instead?

Reply:
It is a space character. We had it before and I'd like to keep it :-)

Reply:
alrigthy then